po/nl.po

```diff
@@ -0,0 +1,1749 @@
+"few. To complement this there is also a built in spectrum analyzer."
+msgstr ""
+"PulseEffecten is een uitgebreid hulpmiddel om audio mee naar uw eigen hand "
+"te zetten. Het bevat een equalizer, limiter, compressor, galmhulpmiddel en "
+"nog een aantal hulpmiddelen. Daarnaast bevat de toepassing ook een "
+"spectrumanalyse-hulpmiddel."
+
+#: data/com.github.wwmm.pulseeffects.appdata.xml.in:16
+msgid ""
+"Because PulseEffects uses the default PulseAudio sound server it will work "
+"with most, if not all, applications you use. All supported applications are "
+"presented in the main window, where each can be enabled individually."
+msgstr ""
+"PulseEffecten maakt gebruik van de PulseAudio-geluidsserver en werkt "
+"daardoor met nagenoeg alle, zo niet alle, toepassingen die u gebruikt. Alle "
+"ondersteunde toepassingen worden getoond in het hoofdvenster, waar elke "
+"toepassing op elk moment kan worden in- of uitgeschakeld."
+
+#: data/com.github.wwmm.pulseeffects.appdata.xml.in:22
+msgid ""
+"Besides manipulating sound output, PulseEffects is able to apply effects to "
+"an input device, such as a microphone. This is, for example, useful in audio "
+"recording, but it also works well during voice conversations."
+msgstr ""
+"Naast het aanpassen van geluidsuitvoer, past PulseEffecten ook het geluid "
+"van invoerapparaten, zoals een microfoon, aan. Dit is onder meer nuttig bij "
+"het opnemen van audio of voeren van audiogesprekken."
+
+#: data/com.github.wwmm.pulseeffects.appdata.xml.in:27
+msgid ""
+"When PulseEffects is launched it will conveniently remember the "
+"configuration used in the last session. It is also possible to save all the "
+"current settings as profiles."
+msgstr ""
+"Na het opstarten toont PulseEffecten de in de vorige sessie gebruikte "
+"instellingen. Ook kunnen instellen worden opgeslagen als profielen."
+
+#: data/com.github.wwmm.pulseeffects.appdata.xml.in:40
+msgid "Set the volume and turn on/off effects"
+msgstr "Regel het volume en schakel effecten in/uit"
+
+#: data/com.github.wwmm.pulseeffects.appdata.xml.in:44
+msgid "Includes an equalizer with built-in presets"
+msgstr "Bevat een equalizer met meegeleverde voorinstellingen"
+
+#: data/com.github.wwmm.pulseeffects.appdata.xml.in:48
+msgid "Input Limiter"
+msgstr "Invoerbeperking"
+
+#: data/com.github.wwmm.pulseeffects.appdata.xml.in:52
+#: data/ui/compressor.glade:110 data/ui/compressor.glade:589
+#: data/ui/webrtc.glade:633
+msgid "Compressor"
+msgstr "Compressor"
+
+#: data/com.github.wwmm.pulseeffects.appdata.xml.in:56
+msgid "Calibration"
+msgstr "Kalibratie"
+
+#: data/com.github.wwmm.pulseeffects.desktop.in:4
+msgid "Equalizer, Compressor and Other Audio Effects"
+msgstr "Equalizer, compressor en overige audio-effecten"
+
+#: data/com.github.wwmm.pulseeffects.desktop.in:6
+msgid "limiter;compressor;reverberation;equalizer;autovolume;"
+msgstr "limiter;compressor;galm;equalizer;automatisch;volume;"
```

Comment:
`automatisch;volume;` should be `autovolume;`?

Reply:
As the idea is controlling the volume automatic it would be better `autovolume` or the equivalent of `automatic volume`

Reply:
You guys are right! I made a tiny error there. I will update the translation and correct this in the meantime.

Reply:
Fixed in #1026 🙂